fspiop-api/documents/API-Definition_v1.1.1.md

```diff
@@ -2264,7 +2264,7 @@ The commit notification is sent in the request **PATCH /transfers/**_{ID}_ from
 
 In case of a validation failure (not schema validation) or another processing error during the processing of a **PUT /transfers/**_{ID}_ callback, a **PATCH /transfers/**_{ID}_**/error** notification can be sent from the Switch to the Payee FSP to indicate of the failure. This notification is sent in cases where the Payee FSP sends the _transferState_ as committed.
 
-[Figure 56a](#figure-56a) shows an example in which the validation in the Switch failed due to some reason, for example the a business rule validation caused a failure in the Switch. This is the same example as in [Figure 53](#figure-53), but the Payee FSP receives a failure notification before performing the actual transfer to the Payee so that the Payee FSP can take remedial action and helps decrease the need for reconciliation between the Switch and the Payee FSP.
+[Figure 56a](#figure-56a) shows an example in which the validation in the Switch failed due to some reason, for example the a business rule validation caused a failure in the Switch. This is the same example as in [Figure 53](#figure-53), but the Payee FSP receives a failure notification so that the Payee FSP can take remedial action (either refund or a similar mechanism).
```

Comment:
This still says "failure notification" (please see my last comment). The Payee FSP would probably not use a refund, rather a reversal (refund is usually initiated by the Payee in a transaction, example a merchant refunds a customer for an earlier purchase, whereas a reversal could be performed by customer care or the system, at least that is how we differentiate between the two).

Reply:
Strange, but yes I get the same (correct version) now..